### DISCUSSION TITLE
Multiple clics on SONOFFS not supported anymore?

### DISCUSSION THREAD

**Discussion by AsWaM** [arendst/Tasmota#15946] [Q&A] [answered]
Hello,

I could not find a breaking change in the documentation, but the double clic on the sxitch is apparently not sent.

Maybe just a payload change?

Used to retrieve it like that 

`{{ trigger.payload_json['TRIG'] == 'DOUBLE' }}`

Thank you

**Comment by sfromis** [comment, marked as answer]
If suspecting a payload change, do use the Tasmota console to watch the payload, to see what you got.

Not using HA, but IIRC, `TRIG` depends on you using `SwitchTopic`

**Comment by AsWaM** [comment]
So i tested, and eventually rolled back the ones i used with the double click. 

There is definitely a change in the way this is handled, the TRIG = DOUBLE is not sent anymore, but in the console i see multipress 1 detected, just can't intercept it apparently.

**Reply by Jason2866** [reply]
From which version did you update? There where changes before v11. Dont remember when exactly.
You have to look in the changelog.
